src/pocketmine/PocketMine.php

```diff
@@ -289,6 +289,10 @@ function server(){
 
 		echo Terminal::$FORMAT_RESET . PHP_EOL;
 
+		if(!flock(\pocketmine\LOCK_FILE, LOCK_UN)){
```

Comment:
It's cleaner to use `fclose()` to avoid mandatory locking problems on Windows. `fclose()` will automatically release locks.

Reply:
No. I tried to replaced the `flock()` with `fclose()` and the resource has not been released. To be sure of closing the resource, I first did the `flock()` and then `fclose()`.

Reply:
hmm, what platform is that on? I was sure fclose releases locks ...

Reply:
![image](https://user-images.githubusercontent.com/14214667/85610015-8303f380-b64e-11ea-87fa-98e9e4e6cb71.png)
never mind, i see now... my mistake